bcgov/mds
# MDS 4202: Ability in CORE to open and view a submitted final IRT

## Objective 

[MDS-4202](https://bcmines.atlassian.net/browse/MDS-4202)

## How to Test

- Navigate to the Project Overview page of a project that has an IRT created
- Using either the IRT tab or the View button on the project stage, navigate to the IRT review screen
- Ensure data matches with what was submitted from Minespace
- Ensure IRT tab and project stages entry are disabled when no IRT exists on the project

## Additional Information / Context 

- Fixed static content for IRT status codes
- Cloned ReviewInformationRequirementsTable from Minespace for display purposes
- Added new IRT tab to Project Overview page
- Added new Final IRT project stage to table
